app/views/users/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1>User#new Form</h1>
+<%# <form method='POST' action='/users' accept-charset="UTF-8"> %>
+<%# <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>
+<%# <label for='usename'>Username: </label> %>
+<%# <input type='text' name='user[username]' placeholder='Enter your username'><br> %>
+<%# <label for='email'>Email: </label> %>
+<%# <input type='email' name='user[email]' placeholder='Enter your email'><br> %>
+<%# <label for='password'>Password: </label> %>
+<%# <input type='password' name='user[password]' placeholder='Enter password at least 8 alphanum'><br> %>
+<%# <input value='submit' type='submit'> %>
+<%# </form> %>
+
+<%# <%= form_tag('/users', method: 'post') do %>
+	<%# <p> %>
+		<%# <%= label_tag(:username, "Username") %> 
+  	<%# <%= text_field_tag(:username) %>
+	<%# </p> %>
+
+  <%# <p> %>
+	 	<%# <%= label_tag(:email, "Email") %>
+		<%# <%= email_field_tag(:email) %> 
+	<%# </p> %>
+
+	<%# <p> %>
+		<%# <%= label_tag(:password, "Password") %> 
+  	<%# <%= password_field_tag(:password) %> 
+	<%# </p> %>
+	  <%# <%= submit_tag("Submit") %> 
+<%# <% end %> 
+
+
+<%= render 'form', user: @user %>
+<a href="/">Go Back</a>
```

Comment:
- You are lacking the form for actually creating the user using `form_for`
Please add a new form for creating a user:
```ruby
<%= form_with @user do |u|%> <%= u.label :username %> <%= u.text_field :username%> 
<%= u.label :email %> 
<%= u.email_field :email %> 
<%= u.label :password %>
<%= u.password_field :password %> 
<%= u.submit %> <% end %>
```